### ISSUE TITLE
Back icon cannot be set for the header

### ISSUE THREAD

**Issue by maks** [Jasonette/JASONETTE-Android#173]
Displaying the back "arrow" icon in the header seems to be automatic functionality on iOS, but on Android this needs to be done explicitly on the toolbar when using a plain Activity subclass.

@gliechtenstein not sure how this should be handled jason syntax wise to keep compatibility with iOS?

**Comment by maks** [comment]
Actually this should more correctly be referring to the "up affordance" per [Android documentation on navigation](https://developer.android.com/design/patterns/navigation.html)

**Comment by gliechtenstein** [comment]
I didn't implement this originally because I think Android's global back button is somewhat equivalent to this feature, and trying to think of a compatible way to express this on ios and android was too much of a headache (not just initially but things like this may introduce unexpected edge cases, which we may need to come back and fix in convoluted ways) 

Plus, I think most Android users are familiar with using the global back button anyway, which is why I just left this as is for now.

But if you have a compelling reason for adding this despite this, please feel free to share. Or if you have a good idea on a JSON spec for this without any headache, that's totally welcome too.

**Comment by realitix** [comment]
I agree with @gliechtenstein on this point. On Android, the global back button seems enough.
